test/access-token.test.js

```diff
@@ -15,6 +15,10 @@ var ds = loopback.createDataSource({ connector: loopback.Memory });
 Token.attachTo(ds);
 var ACL = loopback.ACL;
 
+beforeEach(function() {
+  this.app = app = loopback();
```

Comment:
the mocha-opts requires `support.js` which app ends up global in our test suite 
https://github.com/strongloop/loopback/blob/d85ff32abb681a127659e4ed6ab4cc0fa3fac922/test/support.js#L10-L18